src/org.spec.ts

```diff
@@ -255,4 +255,365 @@ describe('org', () => {
+      });
+      await orgModel.create({
+        name: ORG_NAME,
+        memberAccessType: MemberAccessType.OPEN,
+      });
+
+      const response = await updateOrg(
+        ORG_NAME,
+        {
+          memberAccessType: 'OPEN',
+        },
+        {
+          dbConnection: connection,
+          logger: mockLogging.logger,
+        },
+      );
+
+      expect(response.didSucceed).toBeTrue();
+    });
+
+    test('ASCII Awala endpoint should be allowed', async () => {
+      const connection = getConnection();
+      const orgModel = getModelForClass(OrgModelSchema, {
+        existingConnection: connection,
+      });
+      await orgModel.create({
+        name: ORG_NAME,
+        memberAccessType: MemberAccessType.OPEN,
+        awalaEndpoint: `a.${AWALA_ENDPOINT}`,
+      });
+
+      const response = await updateOrg(
+        ORG_NAME,
+        {
+          awalaEndpoint: AWALA_ENDPOINT,
+        },
+        {
+          dbConnection: connection,
+          logger: mockLogging.logger,
+        },
+      );
+
+      expect(response.didSucceed).toBeTrue();
+    });
+
+    test('non ASCII Awala endpoint should be allowed', async () => {
+      const connection = getConnection();
+      const nonAsciiAwalaEndpoint = 'はじめよう.みんな';
+      const orgModel = getModelForClass(OrgModelSchema, {
+        existingConnection: connection,
+      });
+      await orgModel.create({
+        name: ORG_NAME,
+        memberAccessType: MemberAccessType.OPEN,
+        awalaEndpoint: nonAsciiAwalaEndpoint,
+      });
+
+      const response = await updateOrg(
+        ORG_NAME,
+        {
+          awalaEndpoint: nonAsciiAwalaEndpoint,
+        },
+        {
+          dbConnection: connection,
+          logger: mockLogging.logger,
+        },
+      );
+
+      expect(response.didSucceed).toBeTrue();
+    });
+
+    test('Non ASCII Awala endpoint should be allowed and be updated', async () => {
+      const connection = getConnection();
+      const nonAsciiAwalaEndpoint = 'はじめよう.みんな';
+      const orgModel = getModelForClass(OrgModelSchema, {
+        existingConnection: connection,
+      });
+      await orgModel.create({
+        name: ORG_NAME,
+        memberAccessType: MemberAccessType.OPEN,
+        awalaEndpoint: AWALA_ENDPOINT,
+      });
+
+      await updateOrg(
+        ORG_NAME,
+        {
+          name: ORG_NAME,
+          memberAccessType: 'INVITE_ONLY',
+          awalaEndpoint: nonAsciiAwalaEndpoint,
+        },
+        {
+          dbConnection: connection,
+          logger: mockLogging.logger,
+        },
+      );
+
+      const dbResult = await orgModel.exists({
+        name: ORG_NAME,
+        memberAccessType: MemberAccessType.INVITE_ONLY,
+        awalaEndpoint: nonAsciiAwalaEndpoint,
+      });
+      expect(dbResult).not.toBeNull();
+    });
+
+    test('Malformed name should be refused', async () => {
+      const connection = getConnection();
+
+      const result = await updateOrg(
+        'INVALID_Name',
```

Comment:
I'd factor out  `'INVALID_Name` to a variable just to be explicit about our intent for the two to match.